activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+*   Introduce new database configuration options `keep_alive`, `max_age`, and
```

Comment:
I think you mean `keepalive`

Reply:
nit: tbh I understand why you didn't snake_case it due to precedent on how we use this word in network contexts, but what the setting _does_ here isn't the same as that context, so I think it should be snakecased.

Reply:
From a developer usability perspective it would be nice to use one form or the other consistently.